Auto Bed Levelling with grid / 3 points mix up

Core XY printer, Marlin downloaded 01/03/15. Arduino 1.05

Trying to compile with 3 points method. Turned off grid method. Getting errors on compile:

Marlin_main.cpp: In function 'void process_commands()':
Marlin_main.cpp:1737: error: 'LEFT_PROBE_BED_POSITION' was not declared in this scope
..RIGHT/FRONT/BACK_BED_POSITIONS all not declared.

Marlin Config text below.
# define ENABLE_AUTO_BED_LEVELING
# define Z_PROBE_REPEATABILITY_TEST
# ifdef ENABLE_AUTO_BED_LEVELING

  //#define AUTO_BED_LEVELING_GRID

  #ifdef AUTO_BED_LEVELING_GRID

```
// set the rectangle in which to probe
#define LEFT_PROBE_BED_POSITION 30
#define RIGHT_PROBE_BED_POSITION 150
#define BACK_PROBE_BED_POSITION 150
#define FRONT_PROBE_BED_POSITION 70

#define AUTO_BED_LEVELING_GRID_POINTS 2
```

  #else  // not AUTO_BED_LEVELING_GRID

```
  #define ABL_PROBE_PT_1_X 30
  #define ABL_PROBE_PT_1_Y 70
  #define ABL_PROBE_PT_2_X 30
  #define ABL_PROBE_PT_2_Y 150
  #define ABL_PROBE_PT_3_X 150
  #define ABL_PROBE_PT_3_Y 150
```

  #endif // AUTO_BED_LEVELING_GRID

--- a/Marlin/Marlin_main.cpp
+++ b/Marlin/Marlin_main.cpp
@@ -1734,16 +1734,6 @@ void process_commands()
                 SERIAL_ECHOLNPGM(MSG_POSITION_UNKNOWN);
                 break; // abort G29, since we don't know where we are
             }
-            int left_probe_bed_position=LEFT_PROBE_BED_POSITION;
-            int right_probe_bed_position=RIGHT_PROBE_BED_POSITION;
-            int back_probe_bed_position=BACK_PROBE_BED_POSITION;
-            int front_probe_bed_position=FRONT_PROBE_BED_POSITION;
-            int auto_bed_leveling_grid_points=AUTO_BED_LEVELING_GRID_POINTS;
-            if (code_seen('L')) left_probe_bed_position=(int)code_value();
-            if (code_seen('R')) right_probe_bed_position=(int)code_value();
-            if (code_seen('B')) back_probe_bed_position=(int)code_value();
-            if (code_seen('F')) front_probe_bed_position=(int)code_value();
-            if (code_seen('P')) auto_bed_leveling_grid_points=(int)code_value();
 
 #ifdef Z_PROBE_SLED
             dock_sled(false);
@@ -1764,6 +1754,16 @@ void process_commands()
             feedrate = homing_feedrate[Z_AXIS];
 #ifdef AUTO_BED_LEVELING_GRID
             // probe at the points of a lattice grid
+            int left_probe_bed_position=LEFT_PROBE_BED_POSITION;
+            int right_probe_bed_position=RIGHT_PROBE_BED_POSITION;
+            int back_probe_bed_position=BACK_PROBE_BED_POSITION;
+            int front_probe_bed_position=FRONT_PROBE_BED_POSITION;
+            int auto_bed_leveling_grid_points=AUTO_BED_LEVELING_GRID_POINTS;
+            if (code_seen('L')) left_probe_bed_position=(int)code_value();
+            if (code_seen('R')) right_probe_bed_position=(int)code_value();
+            if (code_seen('B')) back_probe_bed_position=(int)code_value();
+            if (code_seen('F')) front_probe_bed_position=(int)code_value();
+            if (code_seen('P')) auto_bed_leveling_grid_points=(int)code_value();
 
             int xGridSpacing = (right_probe_bed_position - left_probe_bed_position) / (auto_bed_leveling_grid_points-1);
             int yGridSpacing = (back_probe_bed_position - front_probe_bed_position) / (auto_bed_leveling_grid_points-1);

